src/web/modules/gateway_accounts/gateway_accounts.http.ts

```diff
@@ -113,11 +117,15 @@ function getGoLiveUrlForServiceUsingWorldpay(serviceId: string) {
   return `${config.services.SELFSERVICE_URL}/service/${serviceId}/dashboard/live`
 }
 
+function getGoLiveUrlForServiceUsingStripe(gatewayAccountId: string) {
```

Comment:
Could use `external` here to avoid confusion as in many places DB Id is referred to as gatewayAccountId

```suggestion
function getGoLiveUrlForServiceUsingStripe(gatewayAccountExternalId: string) {
```

Reply:
Fixed